### DISCUSSION TITLE
Not possible to update a cart after creating it on a multichannel store?

### DISCUSSION THREAD

**Discussion by ManonSmotly** [bigcommerce/catalyst#1325] [Q&A] [answered]
Hello,

I have an issue with my Catalyst theme plugged on my final store, the cart can be created but not updated. The API request error says:
errors: [
  {
  message: 'Not found: The cart does not exist.',
  path: [ 'cart', 'updateCartLineItem' ],
  locations: [ { line: 3, column: 5 } ]

But I logged in the console the ID of the cart that we are sending in the request. I tried to fetch/update it outside of Catalyst (with the REST API) and the cart exists and can be updated (with the same API scope and tokens that I am using in Catalyst). Also, if I plug the same theme on another store, the cart can be updated and everything works fine.

It seems to be an issue linked to the store config/channels, does anyone knows what situation could prevent the cart from being updated after creation?

Thank you for your help :)

**Comment by matthewvolk** [comment]
👋 Hi @ManonSmotly - does this issue persist if you clear your browser cookies and try with a brand new cart? Recommend using incognito to rule out any browser issues.

**Reply by ManonSmotly** [reply, marked as answer]
Thank you for your answers! For anyone passing by, the mistake was on us: we were using a storefront token instead of a customer impersonation token in our environment. Using a customer impersonation token created with the correct parameters solved the cart issues.
